compute_tools/src/config.rs

```diff
@@ -0,0 +1,47 @@
+use std::fs::{File, OpenOptions};
+use std::io::prelude::*;
+use std::io::{self, Read};
+use std::path::Path;
+
+use anyhow::Result;
+use regex::Regex;
+
+use crate::pg_helpers::PgOptionsSerialize;
+use crate::zenith::ClusterSpec;
+
+/// Check that `line` is inside a text file and put it there if it is not.
```

Comment:
Then why use regex?

Reply:
Refactored it to do a one pass on a file and write at the end if needed